open-sphere-base/core/src/main/java/io/opensphere/core/map/MapManagerMenuProvider.java

```diff
@@ -114,6 +105,64 @@ public int getPriority()
         }
     };
 
+    /**
+     * The popup menu for coordinate information.
+     * 
+     * @param pos the geographic position on the globe.
+     * @return dialog the popup window.
+     */
+    private final JDialog CordMenu(GeographicPosition pos)
+    {
+
+        JDialog dialog = new JDialog();
+        dialog.setTitle("Coordinate Menu");
+        JPanel detailsPanel = new JPanel(new BorderLayout());
+        detailsPanel.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
+
+        DecimalDegrees latitudeDD = Coordinates.create(DecimalDegrees.class, pos.getLat().getMagnitude());
+        DecimalDegrees longitudeDD = Coordinates.create(DecimalDegrees.class, pos.getLon().getMagnitude());
+
+        DegDecimalMin latitudeDDM = Coordinates.create(DegDecimalMin.class, pos.getLat().getMagnitude());
+        DegDecimalMin longitudeDDM = Coordinates.create(DegDecimalMin.class, pos.getLon().getMagnitude());
+
+        DegreesMinutesSeconds latitudeDMS = Coordinates.create(DegreesMinutesSeconds.class, pos.getLat().getMagnitude());
+        DegreesMinutesSeconds longitudeDMS = Coordinates.create(DegreesMinutesSeconds.class, pos.getLon().getMagnitude());
+
+        StringBuilder builder = new StringBuilder("DD:\t");
+        builder.append(latitudeDD.toShortLabelString(14, 6, 'N', 'S').trim()).append("\t");
+        builder.append(longitudeDD.toShortLabelString(14, 6, 'E', 'W').trim()).append("\n");
+
+        builder.append("DMS:\t");
+        builder.append(latitudeDMS.toShortLabelString(14, 6, 'N', 'S').trim()).append("\t");
+        builder.append(longitudeDMS.toShortLabelString(14, 6, 'E', 'W').trim()).append("\n");
+
+        builder.append("DDM:\t");
+        builder.append(latitudeDDM.toShortLabelString(14, 6, 'N', 'S').trim()).append("\t");
+        builder.append(longitudeDDM.toShortLabelString(14, 6, 'E', 'W').trim()).append("\n");
+
+        builder.append("MGRS:\t");
+        MGRSConverter test = new MGRSConverter();
```

Comment:
change name from "test" to something like "converter"

Reply:
Good catch; changed to converter